### DISCUSSION TITLE
中国内地首次使用Github登录一直显示登陆失败

### DISCUSSION THREAD

**Discussion by BOSS2356** [zeabur/.github#32] [Q&A]
第一次使用github登录zeabur，之前一直没用过，反馈链接在内地无法访问，使用邮箱也没用

**Comment by yuaanlin** [comment]
哥们您好，请开启科学上网（全局代理模式）后点击 https://zeabur.com/dc 与我们联系，我们会协助您解决问题，谢谢。
